library_generation/test/resources/integration/google-cloud-java/pr-description-golden.txt

```diff
@@ -0,0 +1,23 @@
+This pull request is generated with proto changes between googleapis commit a17d4caf184b050d50cacf2b0d579ce72c31ce74 (exclusive) and 1a45bf7393b52407188c82e63101db7dc9c72026 (inclusive).
+
+Qualified commits are:
+
+[googleapis/googleapis@7a9a855](https://github.com/googleapis/googleapis/commit/7a9a855287b5042410c93e5a510f40efd4ce6cb1)
+
+[googleapis/googleapis@c7fd8bd](https://github.com/googleapis/googleapis/commit/c7fd8bd652ac690ca84f485014f70b52eef7cb9e)
+
+BEGIN_NESTED_COMMIT
+
+feat: [document-ai] expose model_type in v1 processor, so that user can see the model_type after get or list processor version
+
+PiperOrigin-RevId: 603727585
+
+
+feat: [document-ai] add model_type in v1beta3 processor proto
+
+PiperOrigin-RevId: 603726122
+
+
+feat: Regenerate with the Java code generator (gapic-generator-java) v2.34.0
+
+END_NESTED_COMMIT
```

Comment:
Is this supposed to be coming from 3 commits? If so, the nested commit usage should be something like:
```suggestion
BEGIN_NESTED_COMMIT
feat: [document-ai] expose model_type in v1 processor, so that user can see the model_type after get or list processor version

PiperOrigin-RevId: 603727585
END_NESTED_COMMIT
BEGIN_NESTED_COMMIT
feat: [document-ai] add model_type in v1beta3 processor proto

PiperOrigin-RevId: 603726122
END_NESTED_COMMIT
BEGIN_NESTED_COMMIT
feat: Regenerate with the Java code generator (gapic-generator-java) v2.34.0
END_NESTED_COMMIT
```

Reply:
I've change the format, PTAL.

Reply:
@chingor13 What is the advantage of using three `BEGIN_NESTED_COMMIT` over one `BEGIN_COMMIT_OVERRIDE ` other than keeping the extended description(which would be ignored in the release notes anyway)?